client/payment-details/payment-method/index.js

```diff
@@ -12,72 +12,7 @@ import { Card } from '@woocommerce/components';
 import Loadable from 'components/loadable';
 import PaymentDetailsPaymentMethodDetail from './detail';
 import PaymentDetailsPaymentMethodCheck from './check';
-
-/**
- * Extracts and formats payment method details from a charge.
- *
- * @param {object} charge The charge object.
- * @return {object}       A flat hash of all necessary values.
- */
-const formatPaymentMethodDetails = ( charge ) => {
```

Comment:
Funnily enough, this method completely breaks when `card` is `undefined` or `null` becase where it's called [lower in the code](https://github.com/Automattic/woocommerce-payments/pull/802/files#diff-748a9f38db2411015cb8d669dc01fd09L100-L104) there is no check for whether `card` is defined, and it can in some cases be undefined.

Reimplemented this function in Reason, and there are at least no "Can't find x of undefined" errors anymore 🎉